spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaProducerProperties.java

```diff
@@ -41,6 +43,14 @@ public void setBufferSize(int bufferSize) {
 		this.bufferSize = bufferSize;
 	}
 
+	public int getMaxRequestSize() {
```

Comment:
Besides adding the parameter, we will need to add code in the binder to act on it.